filesystem/node.go

```diff
@@ -74,9 +77,9 @@ func (n Node) Getattr(ctx context.Context, _ *fuse.GetattrRequest, resp *fuse.Ge
 // the directory, Lookup should return ENOENT.
 //
 // Lookup need not to handle the names "." and "..".
-func (n Node) Lookup(ctx context.Context, name string) (fs.Node, error) {
+func (n Node) Lookup(_ context.Context, name string) (fs.Node, error) {
 	if !n.IsDir {
-		return nil, fuse.ENOSYS
+		panic(fmt.Sprintf("Lookup request called on file node: %+v", n))
```

Comment:
Is there no error you can return here? A panic seems dramatic. I would think `syscall.ENOTDIR` would be appropriate.